OneCommunityGlobal/HGNRest
# yihan integrate team code into the Teams page

# Description
<img width="658" alt="截屏2023-09-06 09 22 32" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/94303659/81fd98d4-5a36-43ce-9394-cf826ca59dff">

This is the 3rd PR of "Create editable 4-letter team code option" task.

## Related PRS (if any):
To test this backend PR you need to checkout the [#1266](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1266) frontend PR.

## Main changes explained:
- Update file team.js and teamController.js for adding teamCode property to team object.

## How to test:
1. check into current branch on both frontend and backend.
2. do `npm run build` and `npm start` to run this PR locally
3. follow test steps in [#1266](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1266)
